__tests__/app.test.js

```diff
@@ -85,14 +95,42 @@ describe("get/api/articles/:articleid", () => {
   });
 });
 
-describe("invalid path", () => {
-  test("404 invalid path", () => {
+
+describe("get/api/articles/:articleid/comments", () => {
+  test("Return an array of comments with 200 message for correct article ID", () => {
     return request(app)
-      .get("/invalid-path")
+      .get("/api/articles/3/comments")
+      .expect(200)
+      .then(({ body }) => {
+
+
+        expect(body.article).toEqual([
```

Comment:
If we're expecting to see the comments for the article, really it should be body.comments?